packages/frontend/core/src/components/affine/setting-modal/general-setting/editor/page.tsx

```diff
@@ -33,6 +33,16 @@ export const Page = () => {
       >
         <Switch />
       </SettingRow>
+      <SettingRow
+        name={t[
+          'com.affine.settings.editorSettings.page.display-bi-link.title'
+        ]()}
+        desc={t[
+          'com.affine.settings.editorSettings.page.display-bi-link.description'
+        ]()}
+      >
+        <Switch />
```

Comment:
no impl?

Reply:
After waiting for #7956  to be merged, the specific functions will be implemented in the new PR.